ansible/roles/fanout/templates/arista_7060_deploy.j2

```diff
@@ -121,7 +121,7 @@ interface {{ intf }}
 !
 interface Management 1
  description TO LAB MGMT SWITCH
- ip address {{ device_info[inventory_hostname]["ManagementIp"] }}/{{ hostvars[inventory_hostname].mgmt_subnet_mask_length }}
+ ip address {{ device_info[inventory_hostname]["ManagementIp"] }}
```

Comment:
Need to figure out why this is a prefix in eos template. The value in sonic template is not a prefix.